cfdm/functions.py

```diff
@@ -17,9 +17,25 @@
                __cf_version__,
                __file__)
 
+from .core import DocstringRewriteMeta
+
+from .core.docstring import (_docstring_substitution_definitions
+                             as _core_docstring_substitution_definitions)
+
+from .docstring import _docstring_substitution_definitions
+
 from .constants import CONSTANTS, ValidLogLevels
 
 
+# --------------------------------------------------------------------
+# Merge core and non-core docstring sustituion dictionaries
+# --------------------------------------------------------------------
+_subs = _docstring_substitution_definitions.copy()
+_subs.update(_core_docstring_substitution_definitions)
+_docstring_substitution_definitions = _subs
+del _subs
```

Comment:
It might be worth adding a brief comment explaining why `_docstring_substitution_definitions` can't be updated directly, in case this leaves anyone scratching their heads.

Reply:
Good point. As of Python 3.9 PEP 584  this is much more elegant